src/main/java/com/hydratereminder/HydrateReminderPlugin.java

```diff
@@ -291,12 +294,37 @@ private void handleHydrateNextCommand()
 	 */
 	private void handleHydratePrevCommand()
 	{
-		final Duration timeSinceLastBreak = Duration.between(lastHydrateInstant, Instant.now());
-		final String timeString = getTimeDisplay(timeSinceLastBreak);
-		final String message = timeString + " since the last hydration break.";
+		final Optional<Duration> timeSinceLastBreak = getDurationSinceLastBreak(this.lastHydrateInstant);
+		final String message = formatHandleHydratePrevCommand(timeSinceLastBreak);
 		sendHydrateEmojiChatMessage(ChatMessageType.GAMEMESSAGE, message);
 	}
 
+	/**
+	 * <p>Handle the format of the message of hydrate prev command.</p>
+	 * @param timeSinceLastBreak Optional with duration from last break till now, if not it is empty.
+	 * @return messageFormat generated by handling hydratePrevCommand.
+	 * @since 1.1.2
+	 */
+	protected String formatHandleHydratePrevCommand(Optional<Duration> timeSinceLastBreak) {
+		if(timeSinceLastBreak.isPresent()) {
+			return getTimeDisplay(timeSinceLastBreak.get()) + " since the last hydration break.";
+		}
+		return "No hydration breaks have been taken yet.";
+	}
+
+	/**
+	 * <p>Calculate the duration since the last break</p>
+	 * @param lastHydrateInstant Optional Instant representing the lastHydrateInstant if it exists.
+	 * @return Optional Duration representing duration from last break till now, if not it is empty.
+	 * @since 1.1.2
```

Comment:
Just a small thing, can you change all of the `@since 1.1.2` to `@since 1.2.0`?

Otherwise, all the logic looks good.
I'll test your changes tomorrow and will approve and merge this if it all runs as expected.

Reply:
Hi, the changes are done.